tests/phpunit/elementor/core/utils/test-collection.php

```diff
@@ -469,4 +469,39 @@ public function test_each__breaks_on_false() {
 		// Act
 		$collection->each( [ $mock, 'each_callback' ] );
 	}
+
+	public function test_find() {
+		// Arrange.
+		$collection = Collection::make( [ 'a1', 'b2', 'c3', 'd4' ] );
+
+		// Act.
+		$result = $collection->find( function( $item ) {
+			return strpos( $item, '3' ) !== false;
+		} );
+
+		// Expect.
+		$this->assertEquals( 'c3', $result );
+	}
+
+	public function test_find__with_value() {
+		// Arrange.
+		$collection = Collection::make( [ 'a1', 'b2', 'c3', 'd4' ] );
+
+		// Act.
+		$result = $collection->find( 'b2' );
+
+		// Expect.
+		$this->assertEquals( 'b2', $result );
```

Comment:
hmm, something is weird
shouldn't "find" return a boolean?
I mean... you're looking for a value that you already have
maybe we should have "find" for callback and "contains" for values?

Reply:
Done